Makefile

```diff
@@ -27,6 +27,11 @@ CFLAGS += \
 	-Xclang -disable-llvm-passes \
 	-O2
 
+ifeq ($(shell test $(CLANG_VER) -gt 10; echo $$?),0)
+  CFLAGS += \
+	-Wno-frame-address
+endif
+
```

Comment:
`Wframe-address` was introduced in clang 11 so for the sake of not breaking the default build of clang 8 let's keep this as is.